Emails: Update thank you page in Professional Email

AllTerrainDeveloper opened the pull request:
#### Changes proposed in this Pull Request

- Update header icon
- Update header color
- Update margin color
- Update theme

#### Testing instructions

Visual inspection of the thank you page for Professional Email

_/email/:domain/titan/setup-mailbox/thank-you/:slug_

BEFORE | AFTER
------------ | -------------
![image](https://user-images.githubusercontent.com/5689927/129025023-31e7ae09-be61-41f9-8cd6-5b9ab3e9c3f3.png) | ![image](https://user-images.githubusercontent.com/5689927/129599755-fa8b1f5b-18f5-43f7-a4c5-2a92351f8b91.png)



FIGMA | CURRENT
------------ | -------------
![image](https://user-images.githubusercontent.com/5689927/129025259-a39090f5-ba69-412c-904d-d4901c2c5df7.png) | ![image](https://user-images.githubusercontent.com/5689927/129599587-c4ce2bcd-5493-4ae1-b8c1-32e5d3c5c19b.png)



iPhone X | iPad
------------ | -------------
![image](https://user-images.githubusercontent.com/5689927/129598659-e2ffc800-2fd0-42a7-ad66-30359c566c0c.png) | ![image](https://user-images.githubusercontent.com/5689927/129599510-e395a0ab-d848-4c55-8c1b-ab69759d14cd.png)



Related to {1200182182542585-as-1200709491655472}

AllTerrainDeveloper committed:
Emails: Change colours in header, move theme to titan thank you page, update image in header

AllTerrainDeveloper commented:
The designs provided in Figma contains an illustration that is composed of dozens of small vectors. @saygunnyc should this image also be added and the Flash sale as part of the Thank you details section?

stephanethomas commented on client/my-sites/email/titan-setup-thank-you/index.tsx:
```diff
@@ -22,7 +22,7 @@ import { getSelectedDomain } from 'calypso/lib/domains';
 import { SiteData } from 'calypso/state/ui/selectors/site-data';
 import { ThankYou } from 'calypso/components/thank-you';
-import theme from 'calypso/my-sites/marketplace/theme';
+import theme from './theme';
```
We usually use the full path for imports, even when importing from the same directory:

```suggestion
import theme from 'calypso/my-sites/email/titan-setup-thank-you/theme';
```

This makes it easier to look up those imports in an IDE or code editor. I'm also tempted to think that it also simplifies refactoring with [codemods](https://github.com/facebookarchive/codemod) (we used those a lot in the past).

stephanethomas commented on client/my-sites/email/titan-setup-thank-you/index.tsx:
```diff
@@ -46,7 +46,7 @@ const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
 	const thankYouImage = {
 		alt: translate( 'Thank you' ),
-		src: '/calypso/images/upgrades/thank-you.svg',
+		src: '/calypso/images/illustrations/thank-you-professional-email-logo.svg',
```
May I suggest renaming this new file to `email.svg` or `thank-you-email.svg`? I don't think it should be tied to Titan: we may want to reuse it for Google Workspace, or in other contexts.

stephanethomas commented on client/my-sites/email/titan-setup-thank-you/theme.ts:
```diff
@@ -0,0 +1,37 @@
+	fontSize: {
+		small: string;
+	};
+};
```
Do we really need to introduce those types which duplicate [those ones](https://github.com/Automattic/wp-calypso/blob/df370f92cad143abc77ef605c1c7e7fac2b55334/client/components/thank-you/types.ts#L36-L54)? It looks like the only difference is that the `studioGrey` property was renamed to `background`.

AllTerrainDeveloper replied:
I've reworked the theme object, it should look now clearer and better 🙂

stephanethomas commented on client/my-sites/email/titan-setup-thank-you/theme.ts:
```diff
@@ -0,0 +1,37 @@
+		small: '14px',
+	},
+};
+export default theme;
```
I'm also not sure why we need that particular theme object: I believe you've introduced it to change the background color of the header but as far I can see in mockups and Figma the `Thank You` page for Yoast should look the same.

In a perfect world, we would move all common style settings under `client/components/thank-you` (so that it's easy to apply a change to all `Thank You` page) but I don't not how easy it is to do.

AllTerrainDeveloper replied:
The theme is just a JSON object that it's applied as a prop in the react component we are encapsulating, creating a theme object and attaching to it, we can change some properties by adding a theme provider, instead of creating new stylesheets

daledupreez replied:
I think there are two issues to address here:
1. Using the theme objects at all -- we have them, so we don't need to remove them yet.
2. Using _different themes_ for the two places we use the Thank You components.

I think we should definitely avoid (2) -- we're trying to make these designs look the same across the two use cases. Wherever possible we should be using the _same styles/config/theme_. So for this area, can we move these theme values into the Thank You component as much as possible?

AllTerrainDeveloper replied:
So, do you mean to add as component properties the needed values? Like colors?

daledupreez replied:
My thinking is that we currently have two separate `theme` objects defined, one in `client/my-sites/email/titan-setup-thank-you/theme.ts` and one in `client/my-sites/marketplace/theme.ts`, and we're passing both into the `ThankYou` component.

For now, I think we only want one theme defined in `client/components/thank-you/theme.ts` -- we want to have only one core theme for the Thank You component (at least for now) so we have a consistent experience. We can look at refactoring them out again later if we need more than one theme/style (e.g. for Jetpack thank you screens), but we want WordPress.com to start off consistently.

That said, I haven't looked closely at the elements for a few weeks, so it's possible that we do need two themes for now, but if that is the case, the two themes should _only_ pass in the things that need to be different, and should not replicate anything that should be the same and consistent across the two use cases.

AllTerrainDeveloper replied:
I managed to remove the theming. This makes simpler to call the component, as we don't have to provide a mandatory theme provider. Now, it will have 3 new more props: Background color for header, subtitle, and color for headings (title, subtite) in the header.

Seems to me simpler. Plus they are not mandatory and Yoast thank you page keeps working fine:
![image](https://user-images.githubusercontent.com/5689927/129369298-92529ba8-4609-45f9-b687-63ce5a40de47.png)

stephanethomas reviewed (changes requested):
Thanks for working on this Daniel! A small suggestion for next time, I would recommend adding a screenshot of the whole page - even if the changes only concern a small area - as this gives reviewers (and translators when that applies) more context (and with your awesome Chrome plugin it's very easy to hide personal information anyway). Also a before/after comparison would really help for these types of changes.

Let's see what Saygun thinks of it but I believe this page needs another iteration in order to get closer to the mockup:

![screenshot](https://user-images.githubusercontent.com/594356/128879421-aa1d5d95-d987-4a6c-9f60-92968c75bbec.png)

saygunnyc commented:
Thank you, @AllTerrainDeveloper for tackling this page!
I also appreciate the illustration switch.

> The designs provided in Figma contains an illustration that is composed of dozens of small vectors. @saygunnyc should this image also be added and the Flash sale as part of the Thank you details section?

For emails, I don't believe we need the upsell section on day 1. Though want to note that should be useful if our implementation will also be used by other divisions. cc'ing @JanaMW27 so she's aware her design has moved into development. Also for the upsell for emails, we should think about the upsell itself, and the logic – who sees it, and on what circumstances so I believe we can park that for now. 

> Thanks for working on this Daniel! A small suggestion for next time, I would recommend adding a screenshot of the whole page - even if the changes only concern a small area - as this gives reviewers (and translators when that applies) more context (and with your awesome Chrome plugin it's very easy to hide personal information anyway). Also a before/after comparison would really help for these types of changes.

Thank you, @stephanethomas for this comment. I agree. That would be super helpful for me to understand the state of things better that way. Also, that would be even more amazing if you were to include a mobile screenshot too. I will circulate back to the PR once we have more progress.

Thanks again!

AllTerrainDeveloper committed:
Emails: Thank you component improvements, theme fixes

AllTerrainDeveloper commented:
Reworked Pull Request:

- Better approach to Figma design
- Better handling of theming in thank you component
- Better CSS styling
- Better HTML structure
- Removed unused properties in theme

AllTerrainDeveloper committed:
Emails: Minor adjustments regarding Pull Request.

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -84,10 +99,12 @@ const ThankYouNextStep = ( props: ThankYouNextStepProps ) => {
-			<div>
-				<p>{ stepDescription }</p>
-				<div>{ stepCta }</div>
+			<div className="thank-you__steps">
```
Shouldn't this be `thank-you__step`? This is a single step.
```suggestion
			<div className="thank-you__step">
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -84,10 +99,12 @@ const ThankYouNextStep = ( props: ThankYouNextStepProps ) => {
+					<h3>{ stepTitle }</h3>
+					<p>{ stepDescription }</p>
+				</div>
+				<div className="thank-you__steps-buttons">{ stepCta }</div>
```
As above, I don't think `steps` is correct.

Also, it may be easier to call this `step-cta` to match the incoming variable/prop name.
```suggestion
				<div className="thank-you__step-cta">{ stepCta }</div>
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
-	background-color: var( --studio-gray-0 );
 	display: flex;
 	justify-content: center;
+	background-color: ${ ( props ) => props.theme.colors.backgroundColorHeader };
```
We need to update the Marketplace theme to include these new theme values/properties
_OR_
we should be using one central theme to drive the basic behaviour. And if we're doing that, then it may make sense to just push these values into CSS!

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
+			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
+				<ThankYouFooterContainer>
+					<h1> { thankYouImageFooter } </h1>
+					<h2> { thankYouImageFooterSubtitle } </h2>
+				</ThankYouFooterContainer>
```
I really don't like these names. :-/ I was really confused because I was wondering where you'd added a footer -- I wasn't seeing anything at the bottom of the page.

I think it would be simpler if we refactored the logic here to remove the `ThankYouHeaderTitle` below, and call this `ThankYouTitleContainer`, and then renamed the props to `thankYouTitle` and `thankYouSubtitle`.
```suggestion
				<ThankYouTitleContainer>
					<h1>{ thankYouTitle }</h1>
					{ thankYouSubtitle && <h2>{ thankYouSubtitle }</h2> }
				</ThankYouTitleContainer>
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
 	display: flex;
 	justify-content: center;
+	background-color: ${ ( props ) => props.theme.colors.backgroundColorHeader };
+	height: 352px;
```
I think this should be `min-height` - using `height` means that if we end up with really long text, the text starts to overflow the container.
```suggestion
	min-height: 352px;
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -34,12 +34,27 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
+
+const ThankYouFooterContainer = styled.div< ThankYouThemeProps >`
+	h1 {
+		font-size: ${ ( { subtitle = false } ) => ( subtitle ? '1.5em' : '2em' ) };
```
I think we can remove the `subtitle` argument handling -- this `h1` is always going to be `2em` AFAICT.
```suggestion
		font-size: 2em;
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
-							{ thankYouTitle }
-						</ThankYouHeaderTitle>
-					</ThankYouSectionContainer>
+					{ thankYouTitle && (
+						<ThankYouSectionContainer>
+							<ThankYouHeaderTitle className="thank-you__body-header wp-brand-font">
+								{ thankYouTitle }
+							</ThankYouHeaderTitle>
+						</ThankYouSectionContainer>
+					) }
```
As noted above, let's just move `thankYouTitle` into the blue header. I think we want to move those titles into the header for all use cases.
```suggestion
```

daledupreez commented on client/components/thank-you/style.scss:
```diff
@@ -5,3 +5,15 @@
+
+.thank-you__steps-buttons {
+	margin-left: auto;
+	margin-right: 0;
```
Question: why do we need `margin-right: 0` here? I am not seeing this overriding any values, so I think it's a no-op.

daledupreez commented on client/my-sites/email/titan-setup-thank-you/style.scss:
```diff
@@ -1,3 +1,8 @@
 	margin-left: 4px;
 }
+
+.layout__content {
```
This rule needs to be scoped better. It will apply to the whole UI once this CSS has been loaded, so we can't target just `.layout__content` without additional selectors.

AllTerrainDeveloper replied:
Dale, I am struggling to get a better selector. We need the content layout to have this rule when the thank you page is added, so no borders are shown in the thank you page component.

daledupreez replied:
Hmm... I think we may be able to work around the left and right padding by adding negative margins to the Thank You container. To be specific, at desktop width, it looks like `margin-left: -33px` and `margin-right: 32px` could work.

I haven't had something that has worked to fix the height/bottom area yet, but will keep thinking about it.

AllTerrainDeveloper replied:
That's a clever idea, I didn't think about it. Also I missed this comment 🤦

daledupreez commented on client/my-sites/email/titan-setup-thank-you/theme.ts:
```diff
@@ -0,0 +1,23 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
+	colors: {
+		backgroundColorHeader: string;
+		textHeaderColor: string;
```
Can we rename these to `headerBackgroundColor` and `headerTextColor`? This keeps the naming consistent and makes it easier to start from the element you're interested in.

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
+				<ThankYouFooterContainer>
+					<h1> { thankYouImageFooter } </h1>
```
The `h1` should also have the `wp-brand-font` class applied to make sure we use the correct font. You'll notice the designs are using the same font as the section titles in the page.

daledupreez reviewed (changes requested):
I think this is looking really good!

The only two clear visual issues I had were that the main title isn't using the right font, and the Marketplace/Yoast page doesn't look right because it is not supplying the new theme props. I also noticed a much more subtle issue with the header height with long text.

Most of my comments are much more focused on implementation details, with my biggest concern being the separate theme definitions. For the time being, I think we should push the theme definition within the Thank You code as much as possible, or make it unnecessary if we can.

JanaMW27 commented:
Hello, a couple of comments on my side:
- Can we align the Help links closer to what is in Figma. The Figma new design was intentional to decrease the importance of those buttons and not make them look so much like... buttons.
- On mobile, can we move the CTA below the paragraph on each section as follows:
<img width="358" alt="Screenshot 2021-08-13 at 15 48 25" src="https://user-images.githubusercontent.com/71137829/129368023-bb800800-a192-431a-b605-1bf640bb2fae.png">
So the space and readability work better.
Thank you!

AllTerrainDeveloper commented:
> Hello, a couple of comments on my side:
> 
> * Can we align the Help links closer to what is in Figma. The Figma new design was intentional to decrease the importance of those buttons and not make them look so much like... buttons.
> * On mobile, can we move the CTA below the paragraph on each section as follows:
> 
> <img alt="Screenshot 2021-08-13 at 15 48 25" width="358" src="https://user-images.githubusercontent.com/71137829/129368023-bb800800-a192-431a-b605-1bf640bb2fae.png">
> 
> So the space and readability work better. Thank you!

Hello @JanaMW27 ! 

Thank you for taking the time to add your feedback!

I've changed it a bit so the mobile design is more aligned with your feedback :) 

![image](https://user-images.githubusercontent.com/5689927/129377279-0d028b59-42ff-42ac-aa47-f5baca2ad699.png)

AllTerrainDeveloper committed:
Emails: Pull request suggestions form Dale and Jana

AllTerrainDeveloper committed:
Emails: Fix lint problems

AllTerrainDeveloper commented:
And by applying the heading font style and border colors suggested by Jana, this is how the Thank you page for Professional Emails is:

![image](https://user-images.githubusercontent.com/5689927/129378259-7fbba5d9-d287-4b5a-be79-24a846bd48b6.png)

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -137,11 +113,35 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
-		thankYouImageFooter,
-		thankYouImageFooterSubtitle,
+		headerTextColor,
+		headerBackgroundColor,
```
Nit: Can you move these up the list so they're in alphabetical order?

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -11,19 +11,14 @@ import React from 'react';
-	ThankYouSectionProps,
-	ThankYouThemeProps,
-} from './types';
+import { ThankYouNextStepProps, ThankYouProps, ThankYouSectionProps } from './types';
```
Does it make sense to use `import type` here? That means the type is only imported during TS compilation and not at runtime.
```suggestion
import type { ThankYouNextStepProps, ThankYouProps, ThankYouSectionProps } from './types';
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -151,26 +147,20 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
-				</ThankYouFooterContainer>
+				{ thankYouTitle && (
+					<ThankYouTitleContainer>
+						<h1 className="thank-you wp-brand-font"> { thankYouTitle } </h1>
```
I think we should use a more specific class name for this. I also think we can remove the whitespace outside of the `{}` wrapper.
```suggestion
						<h1 className="thank-you__container_header wp-brand-font">{ thankYouTitle }</h1>
```

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -151,26 +147,20 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
+				{ thankYouTitle && (
+					<ThankYouTitleContainer>
+						<h1 className="thank-you wp-brand-font"> { thankYouTitle } </h1>
+						<h2> { thankYouSubtitle } </h2>
```
Two comments:
1. I think we should make this conditional on `thankYouSubtitle` being defined -- otherwise you end up with `<h2></h2>` in the HTML which can cause spacing trouble due to margin/padding etc for the `h2` element.
2. I'd prefer if we remove the whitespace outside the `{}` wrapper. (This gives `<h2>  </h2>` when it's empty at present.)
```suggestion
						{ thankyouSubtitle && <h2>{ thankYouSubtitle }</h2> }
```

daledupreez commented on client/components/thank-you/types.ts:
```diff
@@ -30,32 +30,8 @@ export type ThankYouProps = {
+	thankYouTitle?: string | TranslateResult;
+	thankYouSubtitle?: string | TranslateResult;
+	headerBackgroundColor: string;
+	headerTextColor: string;
```
Can you please update the Marketplace content to supply these now-required values?

I am also think it may be better to make them both optional and default to the colours we want. (That should be easy to do in the destructuring code.)

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -84,10 +71,12 @@ const ThankYouNextStep = ( props: ThankYouNextStepProps ) => {
+					<h3>{ stepTitle }</h3>
+					<p>{ stepDescription }</p>
+				</div>
+				<div className="thank-you__step-cta">{ stepCta }</div>
```
I think we can remove the `React.Fragment` wrapper now that we're returning a single wrapping `div`. We can shift the `key` prop to the main/root `div`.

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -115,36 +104,63 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
+		thankYouSubtitle,
 		thankYouImage,
+		headerTextColor,
+		headerBackgroundColor,
```
Also, to expand on my comment RE defaulting them in, it may be better to do that here.
```suggestion
		headerTextColor = 'white',
		headerBackgroundColor = '#0675C4',
```

daledupreez commented on client/my-sites/email/titan-setup-thank-you/index.tsx:
```diff
@@ -100,14 +98,18 @@ const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
+			thankYouTitle={ translate( 'Congratulations on your purchase!' ) }
+			thankYouSubtitle={ translate(
+				'You will receive an email confirmation shortly for your purchase.'
+			) }
```
I think we need to remove the subtitle for now and use `Your email is now ready to use!` or `Your mailbox is now ready to use!` for the title.

This is mostly because this flow doesn't involve a purchase -- the user is simply creating their first mailbox.

daledupreez commented on client/my-sites/email/titan-setup-thank-you/style.scss:
```diff
@@ -1,3 +1,8 @@
 	margin-left: 4px;
 }
+
+div.layout__content {
+	padding: 79px 0 32px calc( var( --sidebar-width-max ) );
+	background-color: white;
+}
```
I am thinking we may need to drop these changes from this PR.

The issue with this is that we're still going to match any `div.layout__content` element after we've loaded this CSS, meaning that we're going to get this CSS applied to every page in the UI after someone loads the thank you page. (You can see it in action by clicking on the Manage button from the Thank You page -- the white background is now applied everywhere.)

We _may_ be able to address the left and right padding by making the ThankYou container use margins that are < 0, but I don't think that helps with the bottom of the page.

The tricky issue here is that we need some dedicated way to flag that this content needs to be displayed full-width, and I think that is beyond the scope of this PR.

AllTerrainDeveloper replied:
Thank you Dale, I've applied the negative margin and now it works way better, not changing the parent element.

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -34,16 +29,8 @@ const ThankYouContainer = styled.div< ThankYouThemeProps >`
-
 const ThankYouHeaderTitle = styled.h1`
-	font-size: ${ ( { subtitle = false } ) => ( subtitle ? '1.5em' : '2em' ) };
+	font-size: 2em;
```
I think it may make more sense to call this `ThankYouSectionTitle` as we're no longer using it in the header.

Also, I think this should be 1.5em, as we're only using it within the section content, and this should be the old `subtitle` size.

daledupreez replied:
Also, I am a fan of the cleanup!

daledupreez reviewed (changes requested):
Thanks for folding in the feedback from last week!

One item I didn't see you address was [raised here](https://github.com/Automattic/wp-calypso/pull/55311#issuecomment-898475620):
> Can we align the Help links closer to what is in Figma. The Figma new design was intentional to decrease the importance of those buttons and not make them look so much like... buttons.

I think we can either do that here or in a separate PR, but we should make it clear which approach we want to take!

There are still some issues with the styles and handling in the PR. The biggest issue is still with us targeting `.layout__content` in a way that will bleed into the rest of Calypso. I had one partial thought in [this comment](https://github.com/Automattic/wp-calypso/pull/55311#discussion_r688578591) to address part of that issue.

I also noticed that we need better CSS rules for tablet sizes -- there are borders/padding creeping on for those sizes.

As a final request, once you've wrapped this round of changes, could you update the screenshots? At least some of them seem to be out of date.

AllTerrainDeveloper committed:
Emails: Pull request suggestions

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -8,11 +8,13 @@ import React from 'react';
+	ThankYouNextStepProps,
+	ThankYouProps,
+	ThankYouSectionProps,
+} from 'calypso/components/thank-you/types';
 /**
```
Nit: I think this will be easier to understand/read with a trailing newline.
```suggestion
} from 'calypso/components/thank-you/types';

/**
```

AllTerrainDeveloper replied:
I did that because I had a lint error in the PHP storm, after removing the empty space it disappeared, but now it is not appearing again 😕 . Adding it anyway 🙂

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -160,22 +156,32 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
-							</ThankYouHeaderTitle>
-							<p>
+							</ThankYouSectionTitle>
+							<p className="thank-you__help-title">
```
Nit: I think it would be better to use `thank-you__help-text` as this isn't strictly speaking a title.

daledupreez commented on client/components/thank-you/index.tsx:
```diff
@@ -160,22 +156,32 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
+								target="_blank"
+								rel="noreferrer noopener"
+							>
+								{ translate( 'Support documentation' ) }
```
Nit: the designs have the text here as `'View support documentation'`, which is different than what we had before.

daledupreez commented on client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx:
```diff
@@ -151,6 +151,8 @@ const MarketplaceThankYou = () => {
 				thankYouImage={ thankYouImage }
 				/* TODO: Change thank you message to be dynamic according to product */
 				thankYouTitle={ translate( 'Yoast SEO Premium is installed' ) }
+				headerBackgroundColor={ '#f5f7f7' }
+				headerTextColor={ 'black' }
```
I thought we wanted all these components to use the new colours. The older design had this light grey background, but I think all of the new designs should use the new colour scheme.

Either way, I think the team working on this is going to be updating this component soon, so I'd rather default to the new screen for now, and they can decide otherwise when they get to work on this.

AllTerrainDeveloper replied:
Oh sure, I thought Yoast page had to remain the same as before.

daledupreez commented on client/my-sites/email/titan-setup-thank-you/index.tsx:
```diff
@@ -104,11 +104,6 @@ const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
 			showSupportSection={ true }
 			thankYouImage={ thankYouImage }
 			thankYouTitle={ translate( 'Congratulations on your purchase!' ) }
```
I think this should be `'Your email is now ready to use'` or `'Your email is now ready to use!'`. The copy from the updated designs is intended for normal checkout, while we are showing this screen for the case where a user sets up their mailbox. (This is a tricky one, as we need the old copy and the new design!)

AllTerrainDeveloper replied:
I've reverted it back to the original, which was: _Your email is now ready to use_

daledupreez commented on client/components/thank-you/style.scss:
```diff
@@ -1,7 +1,57 @@
+	color: var( --color-accent );
+}
+
+.thank-you__help-link {
+	display: flex;
+	svg.thank-you__icon-external {
+		height: 16px;
+		width: auto;
+		padding-top: 3px;
+		margin-left: 4px;
+	}
+}
```
I took a quick look at the designs and had two small comments on this:
1. You may want to add some vertical spacing between them using a rule like in the suggestion
2. You may want to look at changing the font size to `$font-body-small`. This doesn't line up 100% with the design, but we should try to stick to standard sizes as much as possible.
```suggestion
.thank-you__help-link {
	display: flex;
	font-size: $font-body-small;

	svg.thank-you__icon-external {
		height: 16px;
		width: auto;
		padding-top: 3px;
		margin-left: 4px;
	}

	+.thank-you__help-link {
		margin-top: 0.25em;
	}
}
```

daledupreez commented on client/my-sites/email/titan-setup-thank-you/style.scss:
```diff
@@ -1,3 +1,13 @@
+
+.thank-you__container {
+	margin: -32px -32px -33px -34px;
+	@media ( max-width: 660px ) {
+		padding: 0 32px 0 34px;
+	}
```
I think we need some padding on the bottom for mobile screens so there is some room below the help links.

daledupreez reviewed (approved):
Overall, this is looking _really_ good! Thanks for working through all the reviews and comments.

My one blocking item is to change the text for the header of the Thank You page -- the user hasn't made a purchase, so we should stick with the older title.

Beyond that I have some small suggestions and non-blocking items, which we can get to in a follow-up PR if needed.

AllTerrainDeveloper committed:
Emails: Amend suggestions from Pull Request.

daledupreez commented:
The updates look good to me! :shipit:

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6446351

Thank you @AllTerrainDeveloper for including a screenshot in the description! This is really helpful for our translators.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6446351).

cpapazoglou commented on client/components/thank-you/index.tsx:
```diff
@@ -115,51 +101,88 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
+		min-height: 352px;
+		img {
+			width: auto;
+			height: 200px;
```
👋  @AllTerrainDeveloper, is there a reason for forcing the img `height`? Also the `width: auto` doesn't help since there is a `max-width: 200px`  here 
https://github.com/Automattic/wp-calypso/blob/70eab0b24c30ac4964bdf4c5e948d21b8404e0f4/client/components/thank-you/style.scss#L5
